Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -676,35 +676,24 @@ static void registerLogHook()
         if (msg->buffer_sz > 1024)
             return;
 
+        // Skip faults and debug logs in non-internal builds.
```

Comment:
Should this be enclosed in `#if USE(APPLE_INTERNAL_SDK)` or something?

Reply:
That is a good point. In this case I do not think it is needed, since this code will only run on non internal builds.